superset-frontend/plugins/plugin-chart-echarts/src/Radar/buildQuery.ts

```diff
@@ -23,8 +23,7 @@ import {
 } from '@superset-ui/core';
 
 export default function buildQuery(formData: QueryFormData) {
-  const { series_limit_metric } = formData;
```

Comment:
I double checked and I think that `timeseries_limit_metric` control is correct.
The problem is in this line - it should be `const { timeseries_limit_metric } = formData;` instead of `const { series_limit_metric } = formData;`.
After I change that name, everything seems to be working fine

Reply:
And that's a good thing, because if we changed the control from `timeseries_limit_metric` to `orderby` we'd also need to add a migration for that control 😄 so fixing the bug here is less work